maintainers/scripts/luarocks-packages.csv

```diff
@@ -39,6 +39,7 @@ lua-cmsgpack,,,,,,
 lua-iconv,,,,,,
 lua-lsp,,,,,,
 lua-messagepack,,,,,,
+lua-protobuf,,,,,,
```

Comment:
here you should add yourself in the maintainer column

Reply:
Hmm, I did in my second commit.

Reply:
you confound with the serpent MR.  update.py takes care of adding the "maintainer" line but to avoid rerunning the script, add it manually:    
`maintainers = with lib.maintainers; [ lockejean ];`

Reply:
Thanks for being patient. Makes total sense. It was just too late yesterday😄🙈

Reply:
should ne fine now